sdk/clientcore/core/src/main/java/io/clientcore/core/util/auth/CompositeChallengeHandler.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package io.clientcore.core.util.auth;
```

Comment:
nit: line break.
moved to its own file?

Reply:
I did this as `ChallengeHandler` was exposing a public `ClientLogger` as interfaces cannot have private constants and the logger was only being used by this inner class.